src/lib/button-toggle/OVERVIEW.md

```diff
@@ -0,0 +1,21 @@
+Buttons toggles aregroups of buttons that can be toggled on and off. They can be configured to 
+behave as checkboxes or as radio buttons. Button toggles are always part of a 
+`md-button-toggle-group`.
```

Comment:
I don't think this is true, you could have a single button that can toggle on or off. The code says the toggle group is optional: https://github.com/angular/material2/blob/master/src/lib/button-toggle/button-toggle.ts#L299